app/definitions/IRocketChat.ts

```diff
@@ -1,5 +1,7 @@
-import { Rocketchat } from '@rocket.chat/sdk';
+import rocketchat from '../lib/rocketchat';
 
-export interface IRocketChat {
-	sdk: typeof Rocketchat;
+type TRocketChat = typeof rocketchat;
+
+export interface IRocketChat extends TRocketChat {
+	sdk: any;
 }
```

Comment:
This change is replicated in other PRs, since the solution applied in dev doesn't work and only then would lint for rocketchat methods work.